frontend/src/components/Apartment/Info.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, List, ListItem, ListItemText, Typography } from '@material-ui/core';
+import React, { ReactElement } from 'react';
+
+type Props = {
+  readonly landlord: string | null;
+  readonly contact: string | null;
+  readonly address: string | null;
+};
+
+const InfoItem = ({ text }: { text: string }) => (
+  <ListItem disableGutters>
+    <ListItemText primary={text} />
+  </ListItem>
+);
+
+export default function Info({ landlord, contact, address }: Props): ReactElement {
+  return (
+    <Box mt={1}>
+      <Typography variant="h5">Info</Typography>
+      <List dense>
+        {landlord && <InfoItem text={`Landlord/Renting Company: ${landlord}`} />}
+        {contact && (
+          <a href={contact}>
```

Comment:
Also, in the future we may want to change the color of the link. We don't have blue anywhere else on the site. It's a question for a designer though, so let's leave it as is for now.